model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -157,6 +185,19 @@ def nodes_builder(model: GraphModule,
         else:
             output_shape = []
 
+        # filter Nodes from framework attr
+        framework_attr_filtered = {}
+        for k, v in framework_attr.items():
+            if not isinstance(v, torch.fx.node.Node):
+                framework_attr_filtered[k] = v
+        framework_attr = framework_attr_filtered
+
+        # filter Nodes from node kwargs
+        node_kwargs = {}
+        for k, v in node.kwargs.items():
+            if not isinstance(v, torch.fx.node.Node):
+                node_kwargs[k] = v
+
```

Comment:
maybe add a comment to explain this code. Say that we replace these attributes with nx graph nodes